docs/Other SDL Features/Setting the Built-in Nav System Destination/index.md

```diff
@@ -1,17 +1,20 @@
-# Setting the Navigation Destination
-Setting a Navigation Destination allows you to send a GPS location, prompting the user to navigate to that location using their embedded navigation. When using the `SendLocation` RPC, you will not receive a callback about how the user interacted with this location, only if it was successfully sent to Core and received. It will be handled by Core from that point on using the embedded navigation system.
+# Setting the Built-In Navigation Destination
+The `SendLocation` RPC gives you the ability to send a GPS location to the embedded navigation app on the head unit. When the request is sent the user will be prompted to navigate to that location using the embedded navigation app. 
 
-!!! note
-This currently is only supported for Embedded Navigation; it does not work with Mobile Navigation Apps at this time.
+When using the `SendLocation` RPC, you will not have access to any information about how the user interacted with this location, only if the request was successfully sent to Core. The request will be handled by Core from that point on using the embedded navigation system.
+
+!!! NOTE
+The send location feature is only supported for the embedded navigation app; it does not work with mobile navigation apps at this time.
 !!!
 
-## Detecting if SendLocation is Available
-To check if `SendLocation` is supported, you may look at `SDLManager`'s `systemCapabilityManager` property after the ready handler is called. Or, you may use `SDLManager`'s `permissionManager` property to ask for the permission status of `SendLocation`.
+## Checking if Send Location is Available
+Since `SendLocation` is a newer RPC, there is a possibility that not all head units will support this request. To check if the request is supported, you may look at the @![iOS]`SDLManager`!@ @![android, javaSE, javaEE]`SdlManager`!@'s `systemCapabilityManager` property after the manager has started successfully.
```

Comment:
The navigation capability is even newer, so your change here won't help them to find out if it's supported or not.

Reply:
Fixed. Added more information to make things clearer